run_route_scripts/run_with_server.py

```diff
@@ -85,6 +85,7 @@ def make_request(post_body):
   parser.add_argument('--output-dir', type=str, help='The directory in which to place the result of each request')
   parser.add_argument('--concurrency', type=int, help='The number of processes to use to make requests', default=multiprocessing.cpu_count())
   parser.add_argument('--format', type=str, help='Supports csv, json, raw and null output formats', default='csv')
+  parser.add_argument('--headers', type=str, help='Additional http headers to send with the requests', action='append', nargs='*')
```

Comment:
Maybe add help text with an example of the expected format - `header_name: header_value` or some such?

Reply:
sure

Reply:
done